README.md

```diff
@@ -139,9 +140,47 @@ If you didn't use Raiden before, you can
 * Learn more by watching explanatory [videos](https://www.youtube.com/channel/UCoUP_hnjUddEvbxmtNCcApg)
 * Read the blog posts on [Medium](https://medium.com/@raiden_network)
 
-### Play with the Raiden Demo dApp
+### Try Out the Raiden Demo dApp
 
-You can play with the Raiden Demo dApp on any Ethereum Testnet hosted at https://lightclient.raiden.network/.
+These step-by-step instructions will guide you through the process for trying the Raiden dApp. We will be using the Ropsten testnet and MetaMask wallet in this example.
+
+__Prerequisites__
+
+You need to have MetaMask installed for your browser.
+
+1. Visit the [MetaMask website](https://metamask.io/) to download and install MetaMask.
+2. Select the Ropsten Test Network when you've logged in to your MetaMask.
+
+
+__Step 1: Acquire and Wrap ETH__
+
+1. Visit the Ropsten faucet at [https://faucet.ropsten.be](https://faucet.ropsten.be).
+2. Enter your keystore address associated with the Ropsten Test Network.
+3. Click the __"Send me test Ether"__ button, you will receive a total of 1 ETH.
+4. Visit [https://0x.org/portal/weth](https://0x.org/portal/weth), this is where you will be wrapping your ETH to WETH.
+5. Click __"Confirm"__ when *0x Poral* asks to connect to your MetaMask.
+6. Click the __"WRAP"__ button in *0x Portal* next to the ETH Token.
+7. Enter the amount of ETH you would like to wrap and click __"Convert"__.
+
+__Step 2: Connect to the Raiden Demo dApp__
+
+1. Visit the Raiden Demo dApp at [https://lightclient.raiden.network/](https://lightclient.raiden.network/).
+2. Click __Connect__ to connect the dApp to your MetaMask.
+3. Select your newly wrapped token in the list of available tokens.
+
+__Step 3: Select a Hub and Open a Channel__
+
+1. Enter ```0x5257964Ef9b81fba7276AF2a97c111AaD7B840D6``` as your hub for testing.
+2. Enter the amount of wrapped ETH (WETH) you want to deposit when opening a channel.
+3. Sign the deposit with your MetaMask.
+4. Click __Open Channel__.
+5. Sign again with your MetaMask when prompted for "Open Channel", "Approve" and "Set Total Deposit".
+
+You can now start making payments. To do so, simply:
+
+1. Enter the address of the receiver of your payment.
```

Comment:
Again, more of a question: Do we have addresses available to send payments to?

Reply:
Not what I am aware of. But I can look into it and if we do or if we set up addresses I can update it later on.